shopfloor_reception_packaging_dimension/services/reception.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.osv import expression
+
+from odoo.addons.base_rest.components.service import to_int
+from odoo.addons.component.core import Component
+from odoo.addons.shopfloor.utils import to_float
+
+
+class Reception(Component):
+    _inherit = "shopfloor.reception"
+
+    @property
+    def dimension_to_update(self):
+        if not hasattr(self.work, "dimension_to_update"):
+            self.work.dimension_to_update = {}
+        return self.work.dimension_to_update
```

Comment:
```suggestion
    def __init__(self, work_ctx):
        super().__init__(work_ctx)
        if not hasattr(self.work, "dimension_to_update"):
            self.work.dimension_to_update = {}
```

Reply:
Changed.
@mt-software-de this @jbaudoux  this change impacts the related extending module, I pushed a fixup on that branch as well.